anilist/src/main/java/com/huchihaitachi/anilist/presentation/AnilistController.kt

```diff
@@ -52,23 +58,32 @@ class AnilistController : Controller(), AnilistView {
 
   override fun render(state: AnilistViewState) {
     state.pageState?.anime?.let(animeEpoxyController::setData)
-    binding.animeListL.pageLoadingPb.visible = state.loading == PAGE
-    binding.animeListL.totalFooterTv.visible = state.loading != PAGE
-    binding.animeListL.totalFooterTv.text = resources?.getString(R.string.total, state.pageState?.anime?.size)
-    binding.animeListL.animeSrl.isRefreshing = state.loading == RELOAD
-    binding.animeListL.errorFooterTv.apply {
+    binding.animeListLayout.pageLoadingPb.visible = state.loading == PAGE
+    binding.animeListLayout.totalFooterTv.visible = state.loading != PAGE
+    binding.animeListLayout.totalFooterTv.text =
+      resources?.getString(R.string.total, state.pageState?.anime?.size)
+    binding.animeListLayout.animeSrl.isRefreshing = state.loading == RELOAD
+    binding.animeListLayout.errorFooterTv.apply {
       visible = state.error != null
       state.error?.let { text = it }
     }
-    bottomSheetBehavior.state = if(state.details == null) {
-      binding.animeListL.animeErv.isEnabled = true
-      binding.animeListL.dimOverlayView.visible = false
+    binding.animeListLayout.dimOverlayView.isClickable = state.details != null
+    bottomSheetBehavior.state = if (state.details == null) {
+      binding.animeListLayout.dimOverlayView.animate()
+        .alpha(0f)
```

Comment:
You can add `if(state.details == null) 0 else 1` here and not duplicate your code

Reply:
fixed